### PR TITLE
feat(install): add -y flag

### PR DESCRIPTION
**Pull request by zlesnr**
Included here is a new command flag which will allow the user to accept all
prompts using default values if none are set in the environment to use instead.

### CODE REVIEW

File: internal/install/execution/go_task_recipe_executor.go [hunk 1]
```diff
@@ -186,37 +186,43 @@ func varsFromRecipe(r types.Recipe) (types.RecipeVars, error) {
...
-			if envConfig.Default != "" {
-				prompt.Default = envConfig.Default
-			}
-
-			result, err := prompt.Run()
-			if err != nil {
-				return types.RecipeVars{}, fmt.Errorf("prompt failed: %s", err)
+		if envValue == "" {
+			if assumeYes {
+				log.Debugf("required env var %s not found, using default value", envConfig.Name)
```
Comment (Julien4218) [inline internal/install/execution/go_task_recipe_executor.go:197]: @zlesnr Is there any issue when the assumeYes is used but a recipe requires an input to be entered and no default is provided?
I believe in this case some sort of error should raised (maybe this is done somewhere else), or alternatively the question should be prompted (perhaps bad given the input said not to?).

Comment (zlesnr) [reply internal/install/execution/go_task_recipe_executor.go:197]: Good callout.  I'll draft a workaround.

Comment (zlesnr) [reply internal/install/execution/go_task_recipe_executor.go:201]: I've updated the code to check for empty default value and return an error if none were found.  How's that?

**Review by sanderblue — DISMISSED** [review]
🐑 it

**Review by sanderblue — APPROVED** [review]
Re 🐑 it

**Pull request merged**